### PR TITLE
release v1.1.0-rc.4

### LINKED ISSUE #2473: release v1.1.0-rc.4
This is a track issue for tidb-operator v1.1.0-rc.4.

previous release issues for 1.1:

- https://github.com/pingcap/tidb-operator/pull/1768
- https://github.com/pingcap/tidb-operator/releases/tag/v1.1.0-beta.1
- https://github.com/pingcap/tidb-operator/pull/1459
- https://github.com/pingcap/tidb-operator/issues/2067
- https://github.com/pingcap/tidb-operator/issues/2195
- https://github.com/pingcap/tidb-operator/issues/2352

Pending PRs:
- [ ] https://github.com/pingcap/tidb-operator/pull/2471

### PR DESCRIPTION
**Pull request by cofyc**
### What problem does this PR solve? 
fixes #2473
### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.1.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+- Support TiCDC in TidbCluster CR. ([#2338](https://github.com/pingcap/tidb-operator/pull/2338), [@weekface](https://github.com/weekface))
+- Update BR to `v3.1.1` in the `tidb-backup-manager` image ([#2425](https://github.com/pingcap/tidb-operator/pull/2425), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Support creating node pools for TiFlash and CDC on ACK ([#2420](https://github.com/pingcap/tidb-operator/pull/2420), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Support creating node pools for TiFlash and CDC on EKS ([#2413](https://github.com/pingcap/tidb-operator/pull/2413), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Expose `PVReclaimPolicy` for `TidbMonitor` when storage is enabled ([#2379](https://github.com/pingcap/tidb-operator/pull/2379), [@Yisaer](https://github.com/Yisaer))
+- Support arbitrary topology-based HA in tidb-scheduler (e.g. node zones) ([#2366](https://github.com/pingcap/tidb-operator/pull/2366), [@PengJi](https://github.com/PengJi))
+- Skip setting the TLS for PD dashboard when the TiDB version is before 4.0.0 ([#2389](https://github.com/pingcap/tidb-operator/pull/2389), [@weekface](https://github.com/weekface))
```
Comment (ran-huang) [inline CHANGELOG-1.1.md:22]:
```suggestion
- Skip setting the TLS for PD dashboard when the TiDB version is earlier than 4.0.0 ([#2389](https://github.com/pingcap/tidb-operator/pull/2389), [@weekface](https://github.com/weekface))
```

File: CHANGELOG-1.1.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+- Support backup and restore with GCS using BR ([#2267](https://github.com/pingcap/tidb-operator/pull/2267), [@shuijing198799](https://github.com/shuijing198799))
+- Update `TiDBConfig` and `TiKVConfig` to support `4.0.0-rc` version ([#2322](https://github.com/pingcap/tidb-operator/pull/2322), [@Yisaer](https://github.com/Yisaer))
```
Comment (ran-huang) [inline CHANGELOG-1.1.md:24]:
```suggestion
- Update `TiDBConfig` and `TiKVConfig` to support the `4.0.0-rc` version ([#2322](https://github.com/pingcap/tidb-operator/pull/2322), [@Yisaer](https://github.com/Yisaer))
```

File: CHANGELOG-1.1.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+- Fix the bug when `TidbCluster` service type is `NodePort`, the value of `NodePort` would change frequently ([#2284](https://github.com/pingcap/tidb-operator/pull/2284), [@Yisaer](https://github.com/Yisaer))
+- Add external strategy ability for `TidbClusterAutoScaler` ([#2279](https://github.com/pingcap/tidb-operator/pull/2279), [@Yisaer](https://github.com/Yisaer))
+- Don't set owner references to keep `PVC` for `TidbMonitor` when it gets deleted ([#2374](https://github.com/pingcap/tidb-operator/pull/2374), [@Yisaer](https://github.com/Yisaer))
```
Comment (ran-huang) [inline CHANGELOG-1.1.md:27]: The release note should be a descriptive sentence, not a demand.

Comment (cofyc) [reply CHANGELOG-1.1.md:27]: `PVC will not be deleted when `TidbMonitor` gets deleted` how about this?

`Don't set owner references` is the implementation details and can be skipped

Comment (ran-huang) [reply CHANGELOG-1.1.md]: LGTM

Comment (cofyc) [reply CHANGELOG-1.1.md]: updated

File: CHANGELOG-1.1.md [hunk 4]
```diff
@@ -1,3 +1,33 @@
...
+- Support backup and restore with GCS using BR ([#2267](https://github.com/pingcap/tidb-operator/pull/2267), [@shuijing198799](https://github.com/shuijing198799))
+- Update `TiDBConfig` and `TiKVConfig` to support the `4.0.0-rc` version ([#2322](https://github.com/pingcap/tidb-operator/pull/2322), [@Yisaer](https://github.com/Yisaer))
+- Fix the bug when `TidbCluster` service type is `NodePort`, the value of `NodePort` would change frequently ([#2284](https://github.com/pingcap/tidb-operator/pull/2284), [@Yisaer](https://github.com/Yisaer))
+- Add external strategy ability for `TidbClusterAutoScaler` ([#2279](https://github.com/pingcap/tidb-operator/pull/2279), [@Yisaer](https://github.com/Yisaer))
+- PVC will not be deleted when `TidbMonitor` gets deleted ([#2374](https://github.com/pingcap/tidb-operator/pull/2374), [@Yisaer](https://github.com/Yisaer))
+- Support scaling for TiFlash ([#2237](https://github.com/pingcap/tidb-operator/pull/2237), [@DanielZhangQD](https://github.com/DanielZhangQD))
+
+
```
Comment (DanielZhangQD) [inline CHANGELOG-1.1.md:30]: The issue in L37 should be #2263

Comment (cofyc) [reply CHANGELOG-1.1.md:30]: fixed

**Comment by cofyc** [comment]
@ran-huang @DanielZhangQD 
all comments are addressed, PTAL

**Pull request merged**